baybe/simulation/core.py

```diff
@@ -34,6 +34,7 @@ def simulate_experiment(
         "error", "worst", "best", "mean", "random", "ignore"
```

Comment:
this PR also deserves a mention int he changelog
tbh I dont see why you see this solution as temporary? I wouldnt mention that, for any given feature theres the option that it will get extended in the future

Reply:
It's really just because, if we added this as default new feature, it would mean:
1. We change the behavior of the function in a technically backward-incompatible way, just only to undo this backward incompatibility like in 2 weeks time when we properly refactor the whole thing
2. Plus, it would then require us again to implement a deprecation mechanism for the publicly exposed flag.

The current version avoids both but unblocks current users.

Reply:
(that's also why I wouldn't even mention in the changelog. it'll go away again with the refactoring)

Reply:
yes i was not suggesting to add a public flag which then has to be deprecated again, that would be major overkill
in fact I suggested to mention in the changelog that `partial results are now returned` or similar (assuming there is no flag as it seems unncecessary to me)
if a future changelog changes this again... isnt that the purpose of the changelog?

Reply:
Ok, I think we are circling back and forth here. How about we agree on the following:
* I'll remove the flag and we make the function simply return incomplete results for now (with changelog entry)
* BUT: also agreeing this will NOT stay forever (i.e. it will have to be replaced with some mechanism that let's the user control the behavior + inspect what when wrong)
* AND: will not deprecate this, i.e. it will become a breaking change in the refactoring, meaning we will have to bump the minor (i.e. middle) version number upon merge

Fair?

Reply:
agree to all + returning a warning when incomplete results are triggered with the inner exception text wouldnt that already solve your `BUT` point?

Reply:
Hi @Scienfitz, I've pushed the corresponding commit. Can you double-check?